src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsPresenter.cs

```diff
@@ -212,5 +212,9 @@ public float GetRegularSnapPoints(Orientation orientation, SnapPointsAlignment a
 
 			return SnapPointsProvider.GetRegularSnapPoints(orientation, alignment, out offset);
 		}
+
+		internal static double OffsetToIndex(double offset) => Math.Max(0, offset - 2);
```

Comment:
What are these cryptic methods for?

Reply:
These methods came with the implementation. We could definitely move them to a helper class instead if they are not meant to be here.

Reply:
Are they actually called anywhere?

Reply:
Yes, both of them:

https://github.com/ajpinedam/uno/blob/dev/add.flipview.support/src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.managed.cs#L545

https://github.com/ajpinedam/uno/blob/dev/add.flipview.support/src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.managed.cs#L892